index.html

```diff
@@ -0,0 +1,806 @@
+            <p><i class="fas fa-tag"></i>Peer-to-Peer, +3</p>
+            <p><i class="fas fa-briefcase"></i>Foundation Capital, +3</p>
+          </div>
+          <div class="funding-section-article-card-footer2">
+            <small>Auxmoney Raises €150M Equity Funding</small>
+            <br>
+            <small class="font-small-3x">on FinSMEs</small>
+          </div>
+        </div>
+        <div class="funding-section-article-cards">
+          <img src="./images/13.jpeg" alt="funds">
+          <div class="funding-section-article-card-header color-white">
+            <h4>Anyvision</h4>
+          </div>
+          <div class="funding-section-article-card-body">
+            <h4>€43M</h4>
+            <small class="font-small-3x">VENTURE CAPITAL</small>
+          </div>
+          <div class="funding-section-article-card-footer">
+            <p><i class="fas fa-tag"></i>Artifical Intelligence, +2</p>
+          </div>
+          <div class="funding-section-article-card-footer2">
+            <small>Anyvision Raises €43M Equity Funding</small>
+            <br>
+            <small class="font-small-3x">on FinSMEs</small>
+          </div>
+        </div>
+        <div class="funding-section-article-cards">
+          <img src="./images/14.jpg" alt="funds">
+          <div class="funding-section-article-card-header color-white">
+            <h4>Mercuryo</h4>
+          </div>
+          <div class="funding-section-article-card-body">
+            <h4>€2.5M</h4>
+            <small class="font-small-3x">SEED FUND</small>
+          </div>
+          <div class="funding-section-article-card-footer">
+            <p><i class="fas fa-tag"></i>Financial Services</p>
+            <p><i class="fas fa-briefcase"></i>Target Global</p>
+          </div>
+          <div class="funding-section-article-card-footer2">
+            <small>Mercuryo.io closes €2.5M Seed Funding</small>
+            <br>
+            <small class="font-small-3x">on FinSMEs</small>
+          </div>
+        </div>
+        <div class="funding-section-article-cards xxl">
+          <img src="./images/14.jpg" alt="funds">
+          <div class="funding-section-article-card-header color-white">
+            <h4>Mercuryo</h4>
+          </div>
+          <div class="funding-section-article-card-body">
+            <h4>€2.5M</h4>
+            <small class="font-small-3x">SEED FUND</small>
+          </div>
+          <div class="funding-section-article-card-footer">
+            <p><i class="fas fa-tag"></i>Financial Services</p>
+            <p><i class="fas fa-briefcase"></i>Target Global</p>
+          </div>
+          <div class="funding-section-article-card-footer2">
+            <small>Mercuryo.io closes €2.5M Seed Funding</small>
+            <br>
+            <small class="font-small-3x">on FinSMEs</small>
+          </div>
+        </div>
+      </section>
+    </section>
+    <section class='dgrid2 article-section-short'>
```

Comment:
In the mobile view, each box should take the full width of the screen.

| Source Page	| Clone Page |
|:---:|:---:|
|![image](https://user-images.githubusercontent.com/57726348/92636582-39de4a00-f2f5-11ea-8362-d42c6b3e769c.png)|![image](https://user-images.githubusercontent.com/57726348/92636606-4793cf80-f2f5-11ea-8376-b22a4b88b985.png)|